client/knowledgebase/components/SearchBar.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+
+export default class SearchBar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSearchInput = this.handleSearchInput.bind(this);
+  }
+
+  handleSearchInput(event) {
+    event.preventDefault();
+    const { onUpdateSearchString } = this.props;
+    onUpdateSearchString(event.target.value);
+  }
+
+  render() {
+    let { searchStr } = this.props;
```

Comment:
These 2 lines should be `const { searchStr = '' } = this.props;`.

Reply:
done